packages/contracts-bridge/contracts/interfaces/IBridgeHook.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.7.6;
+
+interface IBridgeHook {
+    /**
+     * @notice Handles an incoming bridge transfer with some tokens and extra
+     * data. Takes any necessary actions for the hook's purposes.
+     *
+     * This function is intended to allow arbitrary post-bridge actions with
+     * tokens, at a user's discretion. E.g. recollateralize a CDP, exchange for
+     * other tokens, emit an event, etc.
+     *
+     * This hook is called AFTER tokens have been transferred to the hook
```

Comment:
todo: add WARNING

Reply:
add warning to what?

Reply:
sorry this was my brain speaking to itself. 

was thinking it would be prudent to prefix this comment with `WARNING:` or `NOTICE:` or something to draw attention to this for developers

Reply:
think the caps do that pretty well already